test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -85,16 +99,46 @@ verify_release()
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
         echo "Release \"$1\" verifying \"$2\""
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
-        
+
         cd "$1"
         for am in $3; do
-            dir="$2/test/format/RUNDIR.$am"
             echo "$1/wt verifying $2 access method $am..."
-
+            dir="$2/test/format/RUNDIR.$am"
             WIREDTIGER_CONFIG="$EXT" ./wt -h "../$dir" verify table:wt
         done
 }
 
+#############################################################
+# upgrade_downgrade:
+#       arg1: release #1
+#       arg2: release #2
+#       arg3: access methods list
+#############################################################
+upgrade_downgrade()
+{
+        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+        echo "Upgrade/downgrade testing with \"$1\" and \"$2\""
+        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+        # Alternate running each release format test program on the second release build.
+        top="$PWD"
+        for am in $3; do
+            for reps in {1..2}; do
```

Comment:
It's not so intuitive to reason running the loop twice. A comment will be helpful.

Reply:
Added in 74f019c85.